tuxemon/core/item.py

```diff
@@ -172,8 +201,14 @@ class with the name of the effect you define in item.db.
 
         # Loop through all the effects of this technique and execute the effect's function.
         for effect in self.effect:
-            last_effect_name = str(effect)
-            result = getattr(self, last_effect_name)(user, target)
+            effect_lines = str(effect).split()
```

Comment:
I don't like string operations done at runtime that should already be loaded from disk.  I'm not going to block the PR because of this though, since its not in a hot loop, and I don't think this code will be long lived.  Eventually, the effects/actions will be refactored into something better, so its fine here.  An alternative to this would be to create a lightweight "dataclass" style object.  Again, probably not worth it here, but just something to think about.